functional-tests/src/test/tempStorage/test (1).odt

```diff
@@ -0,0 +1 @@
+The quality of mercy is not strained.
```

Comment:
I tip my hat to this

Reply:
It droppeth as the gentle rain from heaven Upon the place beneath.